plugins/node/opentelemetry-instrumentation-aws-sdk/package.json

```diff
@@ -1,83 +1,82 @@
 {
-  "name": "@opentelemetry/instrumentation-aws-sdk",
```

Comment:
This whole file appears to be retabbed with 4 spaces instead of 2

Reply:
Right, my IDE is automatically formatting my files. I'll fix that. Thanks for noticing

Reply:
Fixed